build.env

```diff
@@ -0,0 +1,40 @@
+#
+# build.env
+#
+# Environment file used by scripts and tools. All (default) versions used for
+# building, testing and deploying Ceph-CSI are listed here.
+#
+# The format should be source-able by shell scripts, but do not assume only
+# shell scripts consume this file. Variables that reference variables may not
+# get proporly expanded.
+#
+
+# Ceph version to use
+BASE_IMAGE=ceph/ceph:v15
+CEPH_VERSION=nautilus
+
+# standard Golang options
+GOLANG_VERSION=1.13.9
+GO111MODULE=on
+
+# static checks and linters
+GOLANGCI_VERSION=v1.21.0
+GOSEC_VERSION=2.0.0
+
+# "go test" configuration
+# set to stdout or html to enable coverage reporting, disabled by default
+#TEST_COVERAGE=html
+#GO_COVER_DIR=_output/
+
+# helm chart generation, testing and publishing
+HELM_VERSION=v3.1.2
+
+# minikube settings
+MINIKUBE_VERSION=v1.6.0
+VM_DRIVER=none
+CHANGE_MINIKUBE_NONE_USER=true
+
+# e2e settings
+# - enable CEPH_CSI_RUN_ALL_TESTS when running tests with if it has root
```

Comment:
```suggestion
#  enable CEPH_CSI_RUN_ALL_TESTS when running tests with if it has the root
#   permissions on the host
```

Reply:
I'll phrase it like
```
# - enable CEPH_CSI_RUN_ALL_TESTS when running tests has root
#   permissions on the host (needed for cgroup modifications)
```
The prefix of ` - ` is intentional, there could be other `CEPH_CSI_` variables that affect the test suite (I would like to be able to only test rbd or cephfs). The list format will make it easier for others to add more variables.

Is that ok?